### ISSUE TITLE
Source code for the reference WSD system

### ISSUE THREAD

**Issue by YixingLuan** [SapienzaNLP/mulan#2]
Hello,

Thank you for the interesting work.
Now, I'd like to take the provided MuLaN datasets to reproduce the reported results in the paper. 
It would be great if you could provide the source code for your m-BERT + softmax layer used in your experiments. 
So, I can train the model with the same configurations on MuLaN and replicate the reported results. 

Thank you for your time.
